### PR TITLE
20046 - Account Details missing

### PR DESCRIPTION
**Pull request by sameer0422**
*Issue #:*
https://github.com/bcgov/entity/issues/20046

*Description of changes:* Account Details missing

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2918-0q7f9omc.web.app

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 1]
```diff
@@ -386,24 +396,32 @@ export default class AccountInfo extends Mixins(
...
-        this.isCompleteAccountInfo = false
-        this.errorMessage = this.isAddressEditable
-          ? 'Your account info is incomplete. Please enter your address in order to proceed.'
-          : 'This accounts profile is incomplete. You will not be able to proceed until an account administrator ' +
-            'entered the missing information for this account.'
-        this.$refs.editAccountForm?.validate() // validate form fields and show error message
-        // SBTODO create a method in child comp
-        this.$refs.mailingAddress?.triggerValidate() // validate form fields and show error message for address component from sbc-common-comp
+      const getWarningMessage = (condition: Boolean) => condition
+        ? 'This account info is incomplete. You will not be able to proceed until an account administrator enters the missing information for this account.'
```
Comment (seeker25) [inline auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:414]: Linting issue, break this into two lines

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 2]
```diff
@@ -386,24 +396,32 @@ export default class AccountInfo extends Mixins(
...
+        : 'Your account info is incomplete. Please update any missing account details or address.';
+
+      if (this.isBusinessInfoIncomplete || this.isAddressInfoIncomplete) {
+        this.isCompleteAccountInfo = false;
```
Comment (seeker25) [inline auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:418]: Don't need semicolins

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 3]
```diff
@@ -584,8 +602,10 @@ export default class AccountInfo extends Mixins(
       // Remove Vuex with Vue 3
       if (!(this.isStaff && !this.isStaffAccount)) this.$store.commit('updateHeader')
       this.addressChanged = false
-      if (this.baseAddress) {
+      this.originalAddress = this.currentOrgAddress
+      if (!this.isBusinessInfoIncomplete && !this.isAddressInfoIncomplete) {
         this.isCompleteAccountInfo = true
+        this.warningMessage = '';
```
Comment (seeker25) [inline auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:608]: don't need semi columns

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 4]
```diff
@@ -359,6 +368,7 @@ export default class AccountInfo extends Mixins(
   private originalAddress: Address = null // store the original address..do not modify it afterwards
 
   private errorMessage: string = ''
+  private warningMessage: string = ''
```
Comment (seeker25) [inline auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:371]: convert to composition-api?

Comment (sameer0422) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:371]: Need to create new ticket for it

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue (hunk 1 above)
Comment (sameer0422) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue]: Fixed

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue (hunk 3 above)
Comment (sameer0422) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue]: Fixed

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue (hunk 2 above)
Comment (sameer0422) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue]: Fixed

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2918-0q7f9omc.web.app

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 5]
```diff
@@ -795,4 +813,15 @@ export default class AccountInfo extends Mixins(
...
 }
+.custom-warning {
+  border-radius: 4px;
+  border: 1px solid #FCBA19 !important;
+  background: #FFF7E3 !important;
+  color: #495057;
+}
+::v-deep {
+  .custom-warning .v-icon {
+    color: #F8661A !important;
```
Comment (Jxio) [inline auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:824]: use $app-alert-orange

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 6]
```diff
@@ -795,4 +813,15 @@ export default class AccountInfo extends Mixins(
   text-decoration: underline;
   margin-right: auto;
 }
+.custom-warning {
+  border-radius: 4px;
+  border: 1px solid #FCBA19 !important;
```
Comment (Jxio) [inline auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:818]: theme.scss has: $BCgovGold5: #fcba19;// PRIMARY GOLD

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 7]
```diff
@@ -795,4 +813,15 @@ export default class AccountInfo extends Mixins(
...
+  background: #FFF7E3 !important;
```
Comment (Jxio) [inline auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:819]: use $BCgovGold0

Comment (sameer0422) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue]: Fixed

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue (hunk 6 above)
Comment (sameer0422) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue]: Fixed

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue (hunk 5 above)
Comment (sameer0422) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue]: Fixed

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue (hunk 4 above)
Comment (seeker25) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:371]: ok another PR? Would be nice..  we have 74~ components converted and another 140~ to go

**Review by seeker25 — APPROVED** [review]
Please convert the component to composition-api

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2918-0q7f9omc.web.app

**Pull request merged**